BDPointXamarinBinder.csproj

```diff
@@ -17,7 +17,7 @@
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
     <AndroidClassParser>class-parse</AndroidClassParser>
     <PackageId>Bluedot.PointSDK.Android</PackageId>
-    <PackageVersion>15.3.4.2</PackageVersion>
+    <PackageVersion>15.3.5</PackageVersion>
```

Comment:
Ok, this has been a pretty crazy investigation: turns out, if we use the 15.3.5 packageVersion, the demo app will still be using the 15.3.4 aar. However, if we bump it to 15.3.5.1, or 15.3.6, the demo app correctly uses the 15.3.5 aar, i.e.

1. Build using 17Aug aar, 15.3.5 versionName, app reports 15.3.4.
2. Build using 17Aug aar. 15.3.5.1 versionName, app reports 15.3.5.
3.  Build using 17Aug aar. 15.3.6 versionName, app reports 15.3.5.

I'm still not sure exactly why this may be the case. Can you please verify my findings?